client/src/polyfills.ts

```diff
@@ -63,6 +63,7 @@ import 'zone.js/dist/zone';  // Included with Angular CLI.
 /***************************************************************************************************
  * APPLICATION IMPORTS
  */
+import 'hammerjs';
```

Comment:
If the CPU problem still exists should we just remove this? Are the gesture controls worth it?

Reply:
I was concerned about it at first but then I build website for production and everything was very smooth. So it appears to be a development environment issue only and we can keep this.

Reply:
Sounds good, if it ever is a problem locally we'll yoink it.